common/cpp/react/renderer/components/rnscreens/RNSScreenStackHeaderConfigState.h

```diff
@@ -39,7 +39,7 @@ class JSI_EXPORT RNSScreenStackHeaderConfigState final {
     return MapBufferBuilder::EMPTY();
   };
 #else // ANDROID
-#ifndef NDEBUG
+#if !defined(NDEBUG)
```

Comment:
Just interjecting this in, so that it is aligned with other ifdefs in that file. Let's let it slip this time 🙏🏻